src/gui/optionsdlg.cpp

```diff
@@ -1295,6 +1303,11 @@ void OptionsDialog::enableProxy(int index)
     }
 }
 
+void OptionsDialog::enableIPSubnetWhitelist(bool checked)
```

Comment:
Is this method really needed?

Reply:
It's still needed to disable the Whitelist button when the Subnet Bypass option is initially unchecked upon opening Preferences.

Reply:
I mean why do you need this separate method? Why don't do the following:
```c++
m_ui->IPSubnetWhitelistButton->setEnabled(m_ui->checkBypassAuthIPSubnet->isChecked());
```